Database/Corrections/sodItemFixes.lua

```diff
@@ -35,6 +35,16 @@ function SeasonOfDiscovery:LoadItems()
         [209846] = {
             [itemKeys.objectDrops] = {409692},
         },
+        [210026] = {
+            [itemKeys.objectDrops] = {409942,409949},
+        },
+        [210043] = {
+            [itemKeys.objectDrops] = {410089},
```

Comment:
The item is only dropped by Agon. The quests will though nevertheless.

```suggestion
```

Reply:
Should we then remove the entire entry instead of just the attribute?

Reply:
Yeah, we don't need this correction, as the base file already contains the correct `npcDrops` 👍🏻

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5378/commits/0e47e97e020e00ce5abc0d0ebf34567c00787661